src/components/common-components/autocomplete/autocomplete.component.tsx

```diff
@@ -48,7 +53,7 @@ export function AutocompleteComponent<T>({
         />
       </div>
       {groupedOptions.length > 0 ? (
-        <ul {...getListboxProps()} className="listbox">
+        <ul {...getListboxProps()} className={"listbox " + moreLeftMargin}>
```

Comment:
Tutaj lepiej jest skorzystać z biblioteki, która pozwala na wprowadzenie warunkowych klas styli. Jest ich kilka, ale w naszym projekcie korzystamy z "classNames", np.  możesz to sobie podejrzeć w pliki [button.component.tsx](https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/blob/develop/src/components/common-components/button-component/button.component.tsx)
`        
className={classNames("btn", `btn-${variant}`, disabled ? "btn-disabled" : "", className)}
`

Reply:
Z tym, że w tym przypdaku nie kombinowałbym z tym, a jak coś to poprawiłbym ten ferelny pierwszy dzień.